.buildkite/pipeline.yml

```diff
@@ -107,3 +107,45 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: helm-release
+
+    steps:
+
+      - label: ":helm: release eck-operator helm chart - pipeline upload"
+        # From https://semver.org, prefixed with 'v'.
+        # The '$$' is required when formatting regular expressions: https://buildkite.com/docs/pipelines/conditionals#formatting-regular-expressions
+        if: build.tag != null
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-eck-operator.yml
+
+      - label: ":helm: release eck-operator helm chart manually - pipeline upload"
+        if: build.message == "release eck-operator helm chart"
+        commands:
+          - buildkite-agent pipeline upload .buildkite/pipeline-helm-release-eck-operator.yml
+
+      - label: ":helm: helm eck-resources - Chart.yaml validation check (expected fail when no files have changed)"
```

Comment:
This bothers me.  This seemed to be the `proper` way to do this via BK, but it's always failing in BK (which is 'expected'), which causes github 'checks' to always show failed.  I'm investigating another way to do this, or simply removing this `cd` step for now, and requiring manual release.  Thoughts?

Reply:
removed per conversation with @thbkrkr